netlify.toml

```diff
@@ -6,9 +6,9 @@
   package = "@netlify/plugin-lighthouse"
 
   [plugins.inputs.thresholds]
-    performance = 0.82
+    performance = 0.65
     accessibility = 1
-    best-practices = 0.93
+    best-practices = 0.87
```

Comment:
```suggestion
    best-practices = 0.93
```
Try reverting this change.
If this Best Practices score degradation is [caused by this `Browser errors were logged to the console` error](https://app.netlify.com/sites/wargabantuwarga/deploys/60ea444bbe51b30008a9ccb6), it's a problem that we have to fix because it's not caused by the Google Docs CMS.
It's highly likely to be caused by our Sentry integration changes in this branch.

<img width="777" alt="Screenshot 2021-07-11 at 15 32 56" src="https://user-images.githubusercontent.com/6315466/125186506-45740780-e25d-11eb-933c-8b3d695e11c0.png">